controller.js

```diff
@@ -1,7 +1,89 @@
-var Calculator = require( './Calculator' )
-var clickHandler = document.getElementById( 'calculator-keypad' )
+var Calc = new Calculator() //eslint-disable-line
+var clickElement = document.querySelector( '.calculator-keypad' )
 
-clickHandler.addEventListener( '.click', function( event ) {
-  var value = event.target
-  console.log( event.target )
-} )
+clickElement.addEventListener( 'click', calculatorHandler.bind( this ) )
+document.addEventListener( 'keypress', calculatorHandler.bind( this ) )
+
+function calculatorHandler ( event ) {
+  var displayElement = document.querySelector( '.calculator-display' )
+  var eventKey
+
+  if ( event.key ) {
+    eventKey = parsedKey( event.key )
+  }
+
+  var value = ( !event.key ) ? event.target.value : eventKey
+  var calculatorButton = document.querySelector( `[value='${eventKey}']` )
+
+  if ( eventKey !== null && eventKey ) {
+    calculatorButton.style.backgroundColor = 'black'
+    calculatorButton.style.color = 'white'
+    calculatorButton.style.opacity = '.7'
+    setTimeout( function () {
+      calculatorButton.style.backgroundColor = 'gray'
+      calculatorButton.style.color = 'black'
+      calculatorButton.style.opacity = '1'
+    }, 200 )
+  }
+
+  if ( event.target.value ) {
+    document.querySelector( '[value="="]' ).focus()
```

Comment:
Are you focusing here so that hitting enter will evaluate the equation? Is so just look for `event.key === 'Enter'` in your keyDown handler